doc/source/index.rst

```diff
@@ -31,6 +31,7 @@ Flower's design goals was to make this simple. Read on to learn more.
    fedbn-example-pytorch-from-centralized-to-federated
    recommended-env-setup
    changelog
+   faq-section
```

Comment:
@mariaboerner1987 last change: can we rename this to just `faq` to make the URL nicer?

Reply:
- renamed the file
- change the name in index.rst